### PR TITLE
[BBS-256] Harmonize logging levels

### PR DESCRIPTION
**Pull request by EmilieDel**
Fixes [BBS-256](https://bbpteam.epfl.ch/project/issues/browse/BBS-256).

This PR tries to harmonize the logging levels for every entrypoints.

### CODE REVIEW

**Review by jankrepl — APPROVED** [review]
Looks good:) 
BTW it seems like we are using mostly `INFO` level in our code.

```
$ git grep logger.info | wc -l
     200
$ git grep logger.debug | wc -l
      37
$ git grep logger.warning | wc -l
       7
$ git grep logger.error | wc -l 
       9
```

File: .env.example [hunk 1]
```diff
@@ -15,7 +15,7 @@
 # You should have received a copy of the GNU Lesser General Public License
 # along with this program. If not, see <https://www.gnu.org/licenses/>.
 
-# Integer values of Python loggin levels are as follows (default is WARNING):
+# Integer values of Python logging levels are as follows (default is WARNING):
```
Comment (pafonta) [inline .env.example:18]:
```suggestion
# Integer values of Python logging levels are as follows (default is INFO):
```

Comment (EmilieDel) [reply .env.example]: Good finding! Thanks @pafonta !

**Review by pafonta — CHANGES_REQUESTED** [review]
Oh, sorry, I forgot:

We need to update the `README` too.

Especially the command using `--verbose` and the first part of the NB mentioning it here:
https://github.com/BlueBrain/Search/blob/master/README.md#create-the-mining-cache

**Comment by EmilieDel** [comment]
Really good point @pafonta! I am changing that this afternoon! Thanks!

**Comment by Stannislav** [comment]
I just noticed we have `BBS_MINING_VERBOSE=-vv` in `.env.example`. This might be related to the changes introduced in this PR.

**Comment by EmilieDel** [comment]
Thanks @Stannislav, it is really relevant for this PR. I tried to address your comment in 3bd5d1e.

**Review by pafonta — APPROVED** [review]

**Pull request merged**